lib/streamlit/runtime/pages_manager.py

```diff
@@ -80,7 +80,9 @@ def get_active_script_hash(self) -> "PageHash":
         return self.pages_manager.current_page_hash
 
     def set_active_script_hash(self, _page_hash: "PageHash"):
-        raise NotImplementedError("Unable to set the active script hash in V1 strategy")
+        # Intentionally do nothing as MPA v1 active_script_hash does not
+        # differentiate the active_script_hash and the page_script_hash
+        pass
```

Comment:
I personally do not believe this should be called from the fragment (cause the active script code on init should be the same on rerun in v1), but it seemed safer to remove the associated error here to ensure there's no problems.